base/doc/ltnews40.tex

```diff
@@ -273,6 +273,19 @@ \section{Improvement to \XeTeX\ \cs{showhyphens}}
 %
 \githubissue{1380}
 
+\section{Avoid code duplication in rollback}
+
+When the kernel uses \cs{AddToHook} in a region that might be 
+rolled back (which happens in a few places) and a document requests a
+rollback, then we have the situation that the hook already contains code to which we added same
```

Comment:
?? Probably you meant:
. . . we added the same

Reply:
yes